tests/unittest/test_throttler.py

```diff
@@ -204,6 +245,7 @@ def test_hangid(self, throttler):
         }
         assert throttler.throttle(raw_crash) == (REJECT, "has_hangid_and_browser", 100)
 
+    @freeze_time("2019-05-27 12:00:00", tz_offset=0)
```

Comment:
The infobar test uses a build id, too. We want to make sure that test doesn't get the right answer for the wrong reason.